.gitpod.yml

```diff
@@ -12,6 +12,7 @@ vscode:
     - anweber.statusbar-commands
     - aaron-bond.better-comments
     - James-Yu.latex-workshop
+    - gruntfuggly.triggertaskonsave
```

Comment:
colin:
> optionales, einfach zu aktivierendes aber nicht default
```suggestion
```
hätte den effekt: das feature ist für alle aktiv, die die extension haben. die extension hat man by default nicht
* in vscode einfach per empfehlung installierbar
* in gitpod ~~by default installiert~~ per empfehlung installierbar

einziges problem wäre, wenn jemand die extension hat, aber kein autobuild will. find ich vernachlässigbar

Reply:
Solche Leute werden ignoriert.

Reply:
find ich okay